STUDENT_COURSE _MANAGEMENT/src/main/resources/templates/error.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Error</title>
+</head>
+
+<body>
+
+<h2>Error -Course name already exists</h2>
+
+<hr>
+
+<a th:href="@{/courses/showFormForAdd}" class="btn btn-primary btn-sm mb-3" >
```

Comment:
I think you need to link the bootstrap for the styling!

Reply:
Added the link to bootstrap.